maui-toolkit/Popup/getting-started.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Popup control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
```

Comment:
platform should be "maui-toolkit". 
Refer other controls .md files for common content and do this change